DiscordChatExporter.Core/Exporting/JsonMessageWriter.cs

```diff
@@ -236,6 +236,9 @@ await Context.ResolveAssetUrlAsync(Context.Request.Channel.IconUrl, cancellation
         _writer.WriteString("before", Context.Request.Before?.ToDate());
         _writer.WriteEndObject();
 
+        // Timestamp
+        _writer.WriteString("timestamp", System.DateTime.UtcNow.ToString("yyyy-MM-ddTHH:mm:ss.ffzzz"));
```

Comment:
Can we use .NET's default JSON formatting of dates for this? It should be ISO 8601.

```suggestion
        _writer.WriteString("timestamp", DateTimeOffset.Now);
```

Reply:
And maybe call this something `createdOn` to better convey what this timestamp relates to?

Reply:
You are right about better naming. What about `exportedAt`? In the context of the channel export, `createdAt` may incorrectly suggest channel creation timestamp, not export timestamp.

Reply:
Yeah, that's good